src/eedl/google_cloud.py

```diff
@@ -25,7 +25,7 @@ def get_public_export_urls(bucket_name: str, prefix: str = ""):
 	listing = requests.get(request_url).text
 
 	# comes back as an XML listing - don't need to parse the XML, just need the values of the Key elements
-	pattern = re.compile("\<Key\>(.*?)\<\/Key\>")
+	pattern = re.compile("<Key>(.*?)</Key>")
```

Comment:
This change looks like a mistake to me. We're explicitly searching for angle bracket characters, which have special meaning in the regular expression. We may not need to escape the forward slash, but in many other regexes, we'd need to because they delimit sections. Worth double checking or reverting the changes to the line (which worked last time I tested it).

Reply:
I removed them and, if I'm recalling correctly, tested it and it worked, because PyCharm thinks the '\' are "redundant character escape[s]" Happy to add them back though

Reply:
OK, sounds good - let's leave this as is then.